### PR TITLE
Qthreads merge pre-clean.

### PR DESCRIPTION
**Pull request by gbtitus**
These are minor changes preceding my starting to push the Chapel Qthreads changes back to the Qthreads team.

### CODE REVIEW

**Comment by gbtitus** [comment]
@ronawho , care to take a look at this one?

File: third-party/qthread/Makefile [hunk 1]
```diff
@@ -100,7 +100,7 @@ qthread-config: FORCE
 # ahead in version numbers as that which was used when packaging the
 # Qthreads release
 #
-	cd $(QTHREAD_SUBDIR) && find . -name "*.m4" | xargs touch configure.ac
+	-cd $(QTHREAD_SUBDIR) && find . -name "*.m4" | xargs touch configure.ac
```
Comment (ronawho) [inline third-party/qthread/Makefile]: Is this '-' a typo?

**Comment by ronawho** [comment]
Looks good, modulo the '-' question. I'd imagine that's responsible for the travis build failure since the other changes are just whitespace or comment removal changes.

**Comment by gbtitus** [comment]
The travis build failure seems to be an unrelated chpldoc thing that happened after at least one set of the hello* tests had already passed. I'm inclined to merge this, although I could wait until Thomas had time to look at it and confirm that it was ok. Or maybe I could commit a no-op (ish) change in order to fire the travis check again ...

**Comment by lydia-duncan** [comment]
That looks sufficiently 'not your problem' to me, though I'm sure Thomas will still want to know about it.  In particular, the changes you made would not have affected the chpldoc environment build set up, which is what is failing.

**Comment by ronawho** [comment]
Ah, ok I see what the '-' is for now. I didn't read in the individual commits before (and obviously didn't know what it meant.) 

I think it's ok to merge with how limited the changes are. The regular smoke tests should catch anything

**Pull request merged**